conf/portal.application.conf

```diff
@@ -144,11 +144,25 @@ kairosdb {
 reporting {
   renderers {
     WEB_PAGE {
-      "text/html" {
+      "text/html; charset=utf-8" {
         type = "com.arpnetworking.metrics.portal.reports.impl.chrome.HtmlScreenshotRenderer"
+        chromePath = "/usr/lib/chromium/chrome"
+        # ^ This path is only correct in the MPortal Docker image! If you aren't running MPortal in Docker, and you have reports enabled,
+        #   you'll need to change this to the actual path to your Chrome binary.
+        chromeArgs = {
+          # Command-line flags passed to Chrome. List of valid args: https://peter.sh/experiments/chromium-command-line-switches/
```

Comment:
Are all chrome arguments key-value? Some programs take "flags" as well which have no associated value. How do you represent these here?